### ISSUE TITLE
Issue on launch

### ISSUE THREAD

**Issue by TastyFrenchBun** [ijo42/GLauncherDockered#16]
Heyo!
I have an error when I try to start a container

```
───────────────────────────────────────
launchserver  | GID/UID
launchserver  | ───────────────────────────────────────
launchserver  | **** libs checked ****
launchserver  | 
launchserver  | User UID:    33
launchserver  | User GID:    33
launchserver  | ───────────────────────────────────────
launchserver  | 
launchserver  | [custom-init] No custom files found, skipping...
launchserver  | [ls.io-init] done.
launchserver  | Exception in thread "main" java.lang.reflect.InvocationTargetException
launchserver  |         at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source)
launchserver  |         at java.base/java.lang.reflect.Method.invoke(Unknown Source)
launchserver  |         at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndStartAgent(Unknown Source)
launchserver  |         at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndCallPremain(Unknown Source)
launchserver  | Caused by: java.lang.UnsupportedOperationException: Please remove -javaagent option from start.sh
launchserver  |         at pro.gravit.launchserver.StarterAgent.premain(StarterAgent.java:17)
launchserver  |         ... 4 more
launchserver  | *** java.lang.instrument ASSERTION FAILED ***: "!errorOutstanding" with message Outstanding error when calling method in invokeJavaAgentMainMethod at src/java.instrument/share/native/libinstrument/JPLISAgent.c line: 627
launchserver  | *** java.lang.instrument ASSERTION FAILED ***: "success" with message invokeJavaAgentMainMethod failed at src/java.instrument/share/native/libinstrument/JPLISAgent.c line: 466
launchserver  | *** java.lang.instrument ASSERTION FAILED ***: "result" with message agent load/premain call failed at src/java.instrument/share/native/libinstrument/JPLISAgent.c line: 429
launchserver  | FATAL ERROR in native method: processing of -javaagent failed, processJavaStart failed
launchserver  | Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
launchserver  | V  [libjvm.so+0x9a560b]  jni_FatalError+0x9b
launchserver  | V  [libjvm.so+0xafdeec]  JvmtiExport::post_vm_initialized()+0x23c
launchserver  | V  [libjvm.so+0xf9366b]  Threads::create_vm(JavaVMInitArgs*, bool*)+0x8ab
launchserver  | V  [libjvm.so+0x9a971f]  JNI_CreateJavaVM+0x4f
launchserver  | C  [libjli.so+0x4891]  JavaMain+0x91
launchserver  | C  [libjli.so+0x8719]  ThreadJavaMain+0x9
launchserver  | Aborted (core dumped)

```

docker-compose.yml:
```
version: '2'

services:
  launchserver:
    image: ijo42/glauncher:latest
    tty: true
    stdin_open: true
    mem_limit: 512mb
    container_name: "launchserver"
    ports:
      - "9274:9274"
    volumes:
      - ./launchserver:/app/launchserver
    restart: always
    environment:
      - TZ=Europe/Berlin # Timezone
      - PUID=33  # User  ID `id $(whoami) -u`
      - PGID=33  # Group ID `id $(whoami) -g`
```

I tried image: ijo42/glauncher:master, and to change PUID and PGID to some other users I have, but the same error appeared :(

**Comment by TastyFrenchBun** [comment]
sorry, my mistake, ijo42/glauncher:master actually made the problem go away

**Comment by TastyFrenchBun** [comment]
I have modified Dockerfile and changed the list line to
`CMD ["s6-setuidgid", "abc", "java", "-Dlaunchserver.dockered=true", "-Dlauncher.useSlf4j=true", "-jar", "LaunchServer.jar"]`
So, looks like it's working on latest too now

**Comment by ijo42** [comment]
thx for error message, I'll try to fix it!

**Comment by ijo42** [comment]
closed by 849f1fb
